conda_lock/conda_lock.py

```diff
@@ -284,17 +294,26 @@ def make_lock_files(
         Do not re-solve for each target platform for which specifications are unchanged
     metadata_choices:
         Set of selected metadata fields to generate for this lockfile.
+    with_cuda:
+        The version of cuda requested.
+        '' means no cuda.
+        None will pick a default version and warn if cuda packages are installed.
     metadata_yamls:
         YAML or JSON file(s) containing structured metadata to add to metadata section of the lockfile.
     """
 
-    # initialize virtual package fake
+    cuda_specified = True
```

Comment:
I find the potential future redefinition of `cuda_specified` confusing.

Could we put `cuda_specified = True` in the `if:` clause immediately below?

...